webpack.prod.config.js

```diff
@@ -11,10 +12,6 @@ webpackConfig.plugins = [
         },
     }),
     new webpack.optimize.OccurrenceOrderPlugin(),
-    new webpack.optimize.UglifyJsPlugin({
-        minimize: true,
-        sourceMap: true
-    }),
```

Comment:
Does Webpack handle DCE without the UglifyJsPlugin now?

Reply:
They have removed UglifyJsPlugin entirely, trying to compile with it just throws an error: `Error: webpack.optimize.UglifyJsPlugin has been removed, please use config.optimization.minimize instead.`

Reply:
And minimize is true by default in production mode (https://webpack.js.org/configuration/optimization/#optimizationminimize)

Reply:
Cool, I know they moved it to: https://github.com/webpack-contrib/uglifyjs-webpack-plugin, but looks like it's not needed. :+1: